src/modules/stuyactivities/admin/components/OrgApproval.tsx

```diff
@@ -163,7 +166,12 @@ const OrgApproval = ({
                     <Divider />
 
                     <h4>Keywords</h4>
-                    <p className={"font-mono"}>{org.keywords || "none"}</p>
+                    {!org.keywords && <p>none</p>}
+                    {org.keywords &&
```

Comment:
Could you give each element a key? Are you sure you want to hard code this list of capitalized words?